tests/application/static_site/test_views.py

```diff
@@ -101,6 +105,107 @@ def test_departmental_user_cannot_see_page_unless_shared(test_app_client, logged
     assert page.h1.text.strip() == "Test Measure Page"
 
 
+def test_departmental_user_cannot_see_draft_measure(test_app_client, logged_in_dept_user):
```

Comment:
I think this test is a duplicate of `test_departmental_user_cannot_see_draft()` above?